goon-universe/queries.js

```diff
@@ -241,35 +257,78 @@ function addFinalReflectResponse(studentID, input, promptNum, scenarioID, timest
         .then((succeeded) => callback(succeeded ? "Success!" : ""));
 }
 
+
+// scenarioExists(scenarioID)
+// .then(function(result){
+//     console.log(result);
+// })
+// .catch(function(err){
+//     console.log(err);
+// });
```

Comment:
Are the commented-out functions like this left in on purpose?

Reply:
This comment explains how to use the scenarioExists function that is defined below.